composer.json

```diff
@@ -15,6 +15,7 @@
     ],
     "require": {
         "php": "^5.3.6 || >=7.0 <7.2",
+        "doctrine/annotations": "^1.2",
```

Comment:
this is pitfall of composer. While experiencing conflicts of cli apps I suggest to use phar version.